src/lib.rs

```diff
@@ -1289,7 +1239,7 @@ pub fn verify_contribution(before: &MPCParameters, after: &MPCParameters) -> Res
     }
 
     // None of the previous transformations should change
-    if before.contributions[..] != after.contributions[0..before.contributions.len()] {
+    if before.contributions != after.contributions[..after.contributions.len() - 1] {
```

Comment:
why this index change?

Reply:
I thought that this was more clear that `after_contributions` should be equal to `before.contributions` with an additional contribution appended.

Reply:
that makes sense